docs/other/fontsobject.rst

```diff
@@ -79,6 +79,28 @@ Array of fontObjects; read-only.
 Methods
 =======
 
+.. _FontsObject.fontsWithDefaultDesignAxes:
+
+FontsObject.fontsWithDefaultDesignAxes()
+**********************************************
+
+``app.fonts.fontsWithDefaultDesignAxes()``
+
+**Description**
+
+This function will return an array of :ref:`fontobject` listing all the variable fonts available on the system.
+
+.. code:: javascript
+
+   var variableFontList = app.fonts.fontsWithDefaultDesignAxes()
+   alert(variableFontList.length);
+
+**Returns**
+
+An array of fontObjects.
```

Comment:
John was dinging me about being consistent with these as links/refs. We should just be consistent.